### DISCUSSION TITLE
Error coming in when I integrating Frigate with Home Assistant(Core)

### DISCUSSION THREAD

**Discussion by SOUMYAMAFIA** [blakeblackshear/frigate#12612] [General Support]
![Screenshot 2024-07-25 143943](https://github.com/user-attachments/assets/2eb45a96-f235-4c5c-b8c7-76dddd4a573d)
On the above ss as being seen when I tried to create the alert system by integrating the Frigate with the home assistant core version it shows that no devices and no entities are found whereas when I paste the frigate container url in another web browser then it show perfectly the cameras. I can't understand why this is happening. Looking for a solution.

**Comment by NickM-27** [comment]
the docker address should not be used, instead the host ip address should be used

**Comment by SOUMYAMAFIA** [comment]
but I am using the host ip address not the docker one

**Reply by NickM-27** [reply]
I've never seen a router that uses 172, that is almost always the docker container network IP range. are you sure?

**Comment by SOUMYAMAFIA** [comment]
![Screenshot 2024-07-27 121321](https://github.com/user-attachments/assets/2d217d78-2ae3-4bbe-a087-04845f8fabb2)
see when I tried to run with the router IP address then it shows 'failed to connect'

**Reply by NickM-27** [reply]
You need to use the HA OS host IP address also known as 192.168.0.X where X is the address of the server running frigate.

**Reply by SOUMYAMAFIA** [reply]
Can you tell me where I found the HA OS host IP address?
